tests/test_python3.py

```diff
@@ -26,3 +28,32 @@ def test_python3():
     output_file = open(f"{test_dir}/output.txt", "r")
     assert output_file.readline() == "Success"
     output_file.close()
+
+
+@pytest.mark.integration
+def test_python3_with_docker():
+    """
+    Tests adding dependencies to python3 via a custom docker image
+
+    Specifically tests matrix multiplication via numpy
+    """
+    client = docker.from_env()
+    client.images.build(
+        path="./util/",
```

Comment:
If running the test from anywhere but the directory of the test itself, this fails. For context, the current working directory is wherever pytest is invoked.

Maybe could anchor the relative location to the file itself, a-la https://github.com/trytoolchest/toolchest-client-python/blob/main/toolchest_client/files/tests/test_merge.py#L7?